example-app.conf

```diff
@@ -22,16 +22,18 @@ ELASTICSEARCH_ENDPOINT = 'https://search-api.test.hubmapconsortium.org'
 TYPE_SERVICE_ENDPOINT =  'https://search-api.test.hubmapconsortium.org'
 ENTITY_API_BASE =        'https://entity-api.test.hubmapconsortium.org'
 ASSETS_ENDPOINT =        'https://assets.test.hubmapconsortium.org'
-CELLS_API_ENDPOINT =     'https://cells.test.hubmapconsortium.org/api/'
+XMODALITY_ENDPOINT =     'https://cells.api.hubmapconsortium.org'
```

Comment:
Why are these all the same?  Shouldn't they be TEST, DEV etc.?

Reply:
Eventually... but for now @SFD5311 is deploying straight to the "production" instance, and the other instances are too far behind to be useful for us.

Reply:
Ok!

Reply:
@mccalluc I would recommend recommend keeping the '/api/' at the end of the URL

Reply:
@SFD5311 -- Can you expand on that? We will still be making the same API calls, but this makes the top-level configurations that Zhou sets more consistent. This is somewhat like the Elasticsearch... We always are hitting a particular path under the host, but that's a lower-level detail.

Reply:
Oh, I see.  Yeah, if the intention is to add that '/api/' suffix to XMODALITY_ENDPOINT before passing it to the client, or to tweak the client to add the '/api/' suffix to the end of whatever is passed to it, then that should be fine.  I'd thought that the '/api/' just got lost in translation.  My mistake.